### DISCUSSION TITLE
menuitem external link

### DISCUSSION THREAD

**Discussion by anditsung** [laravel/nova-issues#4604] [Ideas & Feature Requests] [answered]
on menuitem external link will open new tab. 
can add optional variable to disable this behaviour?

my usecase is that i have other app beside nova. and need to access that page without leaving current tab

**Comment by greggh** [comment]
I have this exact use case.

**Comment by mathijsqdrop** [comment]
Would prefer such an option as well!

**Comment by piljac1** [comment]
+1

**Comment by syteks** [comment]
+1

**Comment by richardjkeys** [comment]
+1

**Comment by richardjkeys** [comment]
Hacked around this, bit messy but:

```
MenuItem::externalLink(
                    'Back to App',
                    route('home')
                )->method('POST')
```

Then a route which redirects:

```
Route::post('/home', function (){
        return redirect('/');
    })->name('home');
```

And you need to exclude this route in VerifyCsrfToken.php

**Comment by crynobone** [comment, marked as answer]
You can now specify `openInNewTab()` to open an external link in a separate tab. By default, it will open in the current page starting from `4.13.0`:

```php
MenuItem::externalLink('Documentation', 'https://nova.laravel.com/docs')->openInNewTab();
```

**Comment by forgecodex** [comment]
how to use Action ?
